packages/cli/src/index.ts

```diff
@@ -16,17 +16,26 @@ import './commands/create';
 import pkg from '../package.json';
 
 import { colorPalette } from './constants';
+import { applyConfigToCommander, getCommandAncestry, GLOBAL_OPTION_DEFINITIONS, logActiveConfig, resolveConfig, setActiveConfig } from './config';
 
 export * from './types/storyblok';
 
-dotenv.config(); // This will load variables from .env into process.env
 const program = getProgram();
 
 konsola.br();
 konsola.br();
 konsola.title(` Storyblok CLI `, colorPalette.PRIMARY);
 
-program.option('--verbose', 'Enable verbose output');
+// Set all the global config options
+for (const option of GLOBAL_OPTION_DEFINITIONS) {
+  if (option.parser) {
+    program.option(option.flags, option.description, option.parser, option.defaultValue);
+  }
+  else {
+    program.option(option.flags, option.description, option.defaultValue);
```

Comment:
blocker:

The CI is broken due to the tsc check on the CLI

<img width="1123" height="238" alt="Screenshot 2025-11-11 at 11 47 52" src="https://github.com/user-attachments/assets/adfa25be-a6c1-49e1-81d6-dcb6f01f3dab" />

Try constraining the default generic parameter from `GlobalOptionDefinition` from 'unknown' to the types required by the commander